### ISSUE TITLE
Sometimes [Scan Prices] button shifts down one or two position when pressed.

### ISSUE THREAD

**Issue by uladz** [uladz/PriceTracker#1]
Observed with AwesomeGuildStore addon installed. Happens randomly. When shifts then it can or shift together with [Search] button or just [Scan Prices] leaving a gap between it and [Search]. Need to debug to understand why this is happening.

Price Tracker adds a few buttons to the guild store dialog and for some reason first time you open up the guild store they all become misaligned. Now, the second time (and further more) you open up a guild store then all added buttons are in place and all work as expected. I was debugging Price Tracker yesterday and could not find anything wrong in its implementation really, I suspect that something is not exactly correctly gets initialized and placed in AwesomeGuildStore in the first init. Can you help me to debug this issue on your side, I've poked around your the code but it's quite complex and will take me ages to get to the bottom of this.

**Comment by uladz** [comment]
This is a bug in AwesomeGuildStore actually, I have sent a fix to the author. Waiting for an update.

**Comment by uladz** [comment]
This is a bug in AwesomeGuildStore actually, I have sent a fix to the author. Waiting for an update.

**Comment by uladz** [comment]
Fixed in AwesomeGuildStore v0.31.

**Comment by uladz** [comment]
This is a bug in AwesomeGuildStore actually, I have sent a fix to the author. Waiting for an update.
Fixed in "SearchTabWrapper.lua":
1. There is a bug in SearchTabWrapper:RelocateButtons. It misses every 2nd control it iterates through. You'll see in the code why and how I have fixed it . That's the reason why only when you open AGS several times in a row it finally gets all the buttons right.
2. There is another bug in SearchTabWrapper:AttachButton. It checks for IsHidded && parent:IsHidden but it should rather check for IsControlHidden. PriceTracker creates 2 buttons - one visible and one hidden. If this state is not processed correctly then all controls on AGS panel are shifted 1 position up because SetHeight is not set to 0 correctly. Long story short - this caused a bug that when you press a PriceTracked button then all button would shift one position down because at this moment hidden button would get 0 height correctly and the panel would update.
